README.md

```diff
@@ -2,8 +2,6 @@
 
 <a name="readme-top">Hello Microverse</a>
 
-
-
 <!-- TABLE OF CONTENTS -->
 
```

Comment:
#### I think it would be better to add the missing main headers from the [README template](https://github.com/microverseinc/readme-template) Your README, 😎😎😎

Below 👇  I have highlighted the missing sections in your README 

- [ ] 🛠 Built With
- [ ] 🚀 Live Demo
- [ ] 💻 Getting Started
- [ ] 🔭 Future Features

### Keep it up